instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld

```diff
@@ -0,0 +1,19 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/molecularEntity/GTPDisodiumSalt",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+  "definition": "Guanosine 5'-triphosphate (GTP) sodium salt hydrate.",
```

Comment:
Could we adjust this similarly to the [MgATP instance](https://github.com/openMetadataInitiative/openMINDS_instances/blob/main/instances/latest/terminologies/molecularEntity/MagnesiumATP.jsonld)? 

Suggestion:
"A GTP (guanosine 5'-triphosphate) molecule bonded with two sodium ions (Na+) forming a salt hydrate."

Reply:
done